README.md

```diff
@@ -1,14 +1,15 @@
-[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
+[![CI Status](https://github.com/AY2324S2-CS2103T-W11-1/tp/actions/workflows/gradle.yml/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
 
 ![Ui](docs/images/Ui.png)
 
-* This is **a sample project for Software Engineering (SE) students**.<br>
-  Example usages:
-  * as a starting point of a course project (as opposed to writing everything from scratch)
-  * as a case study
-* The project simulates an ongoing software project for a desktop application (called _AddressBook_) used for managing contact details.
-  * It is **written in OOP fashion**. It provides a **reasonably well-written** code base **bigger** (around 6 KLoC) than what students usually write in beginner-level SE modules, without being overwhelmingly big.
-  * It comes with a **reasonable level of user and developer documentation**.
-* It is named `AddressBook Level 3` (`AB3` for short) because it was initially created as a part of a series of `AddressBook` projects (`Level 1`, `Level 2`, `Level 3` ...).
-* For the detailed documentation of this project, see the **[Address Book Product Website](https://se-education.org/addressbook-level3)**.
-* This project is a **part of the se-education.org** initiative. If you would like to contribute code to this project, see [se-education.org](https://se-education.org#https://se-education.org/#contributing) for more info.
+# Welcome to **CareerSync!**
+## This is a CLI-managed internship application tracker that satisfies all your basic needs.
+
+Effortlessly manage, search, and sift through your various internship applications. Enter details rapidly using CLI, and
+avoid losing track of crucial information. Targeted to those with numerous applications to keep track of and prefer 
+using CLI. Your all-in-one solution for seamless application management.
+
+* For the detailed documentation of this project, see the **[CareerSync Product Guide](docs/Documentation.md)**.
+* For details of our members who developed this app, see the **[About Us](docs/AboutUs.md)**.
```

Comment:
Your links here are actually local links, so when I tested it it just leads me to the github link containing that file.

Perhaps you wanted to link it to the actual site?
Our site is hosted here, but it's actually not updated yet

https://ay2324s2-cs2103t-w11-1.github.io/tp/DeveloperGuide.html
https://ay2324s2-cs2103t-w11-1.github.io/tp/AboutUs.html